tracing-attributes/src/attr.rs

```diff
@@ -177,7 +177,7 @@ impl Parse for EventArgs {
                     match ident.to_string().as_str() {
                         "Debug" => result.mode = FormatMode::Debug,
                         "Display" => result.mode = FormatMode::Display,
-                        "Raw" => result.mode = FormatMode::Raw,
+                        "Raw" => result.mode = FormatMode::Value,
```

Comment:
this should also be changed to `Value` --- it looks like right now, we've renamed the macro's internal representation, but not the actual parsed syntax:
```suggestion
                        "Value" => result.mode = FormatMode::Value,
```

Reply:
Thanks, I think this was already fixed in some of the later commits 😄.